packages/sessions-sdk-react/src/components/token-amount-input.tsx

```diff
@@ -12,11 +12,20 @@ export const TokenAmountInput = ({
   max,
   gt,
   lt,
+  onValidationChange,
```

Comment:
You aren't using this in the end and there was some indication that it caused render issues so we should revert this until we go to revisit down the line